.gitignore

```diff
@@ -217,10 +217,12 @@
 /lib/evc_armv4_lib
```

Comment:
BTW, does "evc" stand for Embedded Visual C++? If so, the two evc entries should be removed.

Reply:
Yes, you're right, I'll do it, thanks.